api_metadata_test.go

```diff
@@ -0,0 +1,237 @@
+package blockfrost_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/blockfrost/blockfrost-go"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResourceMetadataTxLabels(t *testing.T) {
+	t.Parallel()
+
+	expectedElement := blockfrost.MetadataTxLabel{
+		Label: "1967",
+		Cip10: "nut.link metadata oracles registry",
+		Count: "3",
+	}
+	expectedElementNull := blockfrost.MetadataTxLabel{
+		Label: "1990",
+		Count: "1",
+	}
+	s := httptest.NewServer(
```

Comment:
Switch to testing on live server and generating/updating golden files for endpoints don't change frequently. Move json data to `testdata/json/{resource}/somename.json` for use with httptest and unmarshall tests. See implmentation in `https://github.com/blockfrost/blockfrost-go/blob/master/api_addresses_test.go`

Reply:
OK